src/app/workspace/organization/users/user-filter/organization-user-filter.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="form-control d-flex p-0 m-2 filter">
+  <input class="border-0 flex-grow-2 px-2" placeholder="Filter users"/>
+
+  <button class="btn search-button"
+          type="button"
+          title="Filter"
+          i18n-title="@@organizations.users.filter">
+    <i class="fa fa-search" aria-hidden="true"></i>
+    <span class="d-none" i18n="@@organizations.users.filter.button">Filter</span>
+  </button>
```

Comment:
Cannot we reuse [SearchButtonComponent](https://github.com/Lumeer/web-ui/blob/devel/src/app/shared/search-box/button/search-button.component.html) here so we don't write the same component twice and have unified styles across the application?

Reply:
Only if I can modify the component.
As is.. it won't be able to filter users just by typing text into it.

Reply:
I'm talking about reusing just the button not the whole search bar. It should be easy, right?

Reply:
It is.

Reply:
I'm just wondering if we really need that button here if you want to implement filtering just by typing the text. What do you think? What is the purpose of having the button here then?

Reply:
There is none.
It was copied from search bar.
I'll make it just a nonclickable icon.